src/react-components/remix-scene-dialog.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { FormattedMessage } from "react-intl";
+
+import styles from "../assets/stylesheets/remix-scene-dialog.scss";
+import DialogContainer from "./dialog-container";
+import { getPlatform, fetchRelease, releasesLink } from "../utils/spoke-download-link";
+
+const Steps = {
+  DownloadSpoke: "download-spoke",
+  DownloadScene: "download-scene",
+  OpenScene: "open-scene"
+};
+
+export default class RemixSceneDialog extends Component {
+  static propTypes = {
+    sceneUrl: PropTypes.string,
+    onClose: PropTypes.func
+  };
+
+  state = {
+    step: Steps.DownloadSpoke,
+    platform: getPlatform(),
+    downloadUrl: releasesLink
```

Comment:
i'd call this `downloadSpokeUrl` to avoid confusing between this and the project download url